package.json

```diff
@@ -18,9 +19,11 @@
   },
   "homepage": "https://github.com/pelias/query#readme",
   "dependencies": {
-    "check-types": "^7.0.0"
+    "check-types": "^7.0.0",
+    "jshint": "^2.9.2"
```

Comment:
This is actually a carry over from https://github.com/pelias/query/pull/19, not sure how it ended up in both places.